adcc/ExcitedStates.py

```diff
@@ -657,6 +659,48 @@ def describe_amplitudes(self, tolerance=0.01, index_format=None):
                 ret += separator
         return ret[:-1]
 
+    def to_dataframe(self, export_vectors=True):
+        """
+        Exports the ExcitedStates object as :class:`pandas.DataFrame`.
+
+        Parameters
+        ----------
+        export_vectors: bool, optional
+            Whether to export vector quantities (transition moments)
+        """
+        propkeys = extract_excitation_property_keys(self)
+        propkeys.extend(self.excitation_energy_corrections.keys())
+        data = {
+            "excitation": np.arange(0, self.size, dtype=int),
+            "kind": np.tile(self.kind, self.size)
+        }
+        # fill the dict with data, sort out stuff that we cannot represent
+        # in a table
+        from adcc import Tensor, AmplitudeVector, OneParticleOperator
```

Comment:
Not a fan of this. Why not just `import adcc` at the top (or sth like that) and then `adcc.Tensor` etc. ?

Reply:
Yeah, that's nicer!

Reply:
Done.